test/integration/targets/azure_rm_rediscache/tasks/main.yml

```diff
@@ -190,9 +191,115 @@
       - facts.rediscaches[0].subnet != None
       - facts.rediscaches[0].access_keys.primary != None
 
-# - name: Delete the redis cache
-#   azure_rm_rediscache:
-#     resource_group: "{{ resource_group }}"
-#     name: "{{ redis_name }}2"
-#     state: absent
-#   register: output
\ No newline at end of file
+- name: Wait for Redis provisioning to complete
+  azure_rm_rediscache_info:
+      resource_group: "{{ resource_group }}"
+      name: "{{ redis_name }}2"
+  register: facts
+  until: "{{ facts.rediscaches[0]['provisioning_state'] == 'Succeeded' }}"
+  retries: 30
+  delay: 60
+  tags:
+    - long_run
```

Comment:
The `never` tag is also needed. Otherwise the task still runs by default.

The same applies for all the other tasks with `long_run`.

Reply:
sure. will link pr here when ready.

Reply:
Matt fixed it already. thanks Matt.